style.css

```diff
@@ -68,8 +96,164 @@ ul {
 
 #menu-button {
   width: 13%;
+  height: 13%;
+}
+
+.works {
+  grid-template-rows: repeat(4, 76.8vw);
+  grid-template-columns: 87vw;
+  grid-row-gap: 12.25vw;
+  margin: 31vw 6.4vw;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #dfe1e6;
+  border-radius: 4.26vw;
+  margin-bottom: 21.6vw;
+}
+
+.card-item {
+  margin: 2% 4%;
+}
+
+.card-img {
+  width: 93%;
+  align-self: center;
+  margin-top: 4%;
+}
+
+h3 {
+  font-size: 8.53vw;
+  font-weight: 700;
+  color: #172b4d;
+}
+
+.company {
+  margin-top: 0;
+  display: flex;
+  justify-content: flex-start;
+  align-items: baseline;
+}
+
+h4 {
+  font-weight: 600;
+  font-size: 3.46vw;
+  color: #344563;
+  margin: 0;
+}
+
+.canopy {
+  margin-left: 0;
+}
+
+.company li p {
+  font-size: 3.46vw;
+  color: #7a869a;
+  margin: 0;
+}
+
+.company li {
+  margin: 2.13vw 1.7vw;
+}
+
+.categories {
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+  align-items: baseline;
+}
+
+.categories li {
+  background-color: #ebebff;
+  border-radius: 2.13vw;
+  margin: 0 1.75vw;
+}
+
+.about-myself {
+  border-top-right-radius: 21.3vw;
+  background-color: #fff;
+}
+
+.about-myself h3 {
+  margin-top: 0;
+  padding-top: 30.4vw;
+  margin-bottom: 0;
+}
+
+.about-myself p.bio {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 4.26vw;
+  line-height: 6.4vw;
+  color: #344563;
+  margin-top: 3.2vw;
+  margin-bottom: 0;
+}
+
+li.abilities {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  border-style: none none solid;
+  border-width: 0.3vw;
+  border-color: #dfe1e6;
+}
+
+img.next {
+  width: 7vw;
+  height: 7vw;
+  margin-right: 11.2vw;
+}
+
+li.languages {
+  display: flex;
```

Comment:
- [ ] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` and assigning it wherever possible as a class name in your `html` markup code.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```
- And kindly attend to any repetitions in your code 🙂.

Reply:
Noted! I'll be very carefull not to break my code while doing so, thanks you for pointing this out!

Reply:
Issue addressed, I refactored the code! 👨‍💻